backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -209,34 +259,29 @@ public Project insertProject(Project project, UserInfo userInfo)
       LOGGER.debug("Project role bindings created successfully");
       transaction = session.beginTransaction();
       projectEntity.setCreated(true);
+      projectEntity.setVisibility_migration(true);
```

Comment:
Unrelated

Reply:
The New visibility project does not need to migrate again so It will protect to fetch the new visibility project created while running the migration. 
Unrelated creating separate PR for this.